securedrop/source_templates/index.html

```diff
@@ -62,7 +62,7 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
+      <p>You appear to be using the Tor Browser. In Tor's "Security Settings" pane (accessible from the toolbar's onion-button), we reccommend turning Security Slider to High—it's easy-peasy!</p>
```

Comment:
> and I kinda intended it to target 0.3.11, was that bad?

We're frozen on the features we we want to land in `0.3.11`, so I assume @redshiftzero was targeting develop to make sure your changes still get included in the near future, even though the release ship has sailed on 0.3.11. We just got through final QA on that (see #1563).

Looks like the automatic branch retargeting via GitHub made some ugly decisions, so we'll have to cherry-pick your contributions and resubmit a new PR that supersedes this one. Smoke and mirrors, largely—we'll take care of the dirty work there. 

Your UX feedback continues to enlighten, thanks very much for your time!